scrape/target.go

```diff
@@ -420,10 +428,18 @@ func (app *maxSchemaAppender) AppendHistogram(ref storage.SeriesRef, lset labels
 	return ref, nil
 }
 
-// PopulateLabels builds a label set from the given label set and scrape configuration.
-// It returns a label set before relabeling was applied as the second return value.
-// Returns the original discovered label set found before relabelling was applied if the target is dropped during relabeling.
-func PopulateLabels(lb *labels.Builder, cfg *config.ScrapeConfig) (res, orig labels.Labels, err error) {
+func PopulateDiscoveredLabels(lb *labels.Builder, cfg *config.ScrapeConfig, tlset, tgLabels model.LabelSet) {
```

Comment:
ditto, is there a reason for `tlset` vs tgLabels` naming inconsistency? both are labels? one from target (or sd?) one from (target group)? 🤔

Reply:
I read it as 't' for target and 'tg' for target group.

Reply:
Sorry I missed your question.  `tlset` was a name previously in `TargetsFromGroup`.